drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -1132,8 +1164,11 @@ local function device_added(driver, device)
     device:emit_event(capabilities.energyMeter.energy({ value = 0.0, unit = "Wh" }))
   end
 
-  -- call device init in case init is not called after added due to device caching
-  device_init(driver, device)
+  -- when unit testing, call set_configure elsewhere
+  if not device:get_field(TEST_CONFIGURE) then
+    -- call device init in case init is not called after added due to device caching
```

Comment:
I don't believe TEST_CONFIGURE should be necessary in this case. Please see [this test file](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/blob/57b46bff0a96a52f868bf6c7d84daffc66a7ea36/drivers/SmartThings/matter-switch/src/test/test_matter_multi_button_switch_mcd.lua) for an example how to test without this extra logic inserted.